egov-persister/case-persister.yml

```
@@ -381,6 +417,60 @@ serviceMaps:
               type: JSON
               dbType: JSONB
 
+        - query: INSERT INTO dristi_case_poaholders(id,tenant_id,case_id,individual_id,poa_type,name,is_active,additional_details,hasSigned,representing_litigants,created_by,last_modified_by,created_time,last_modified_time) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?) ON CONFLICT (id) do update set 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)